kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/position/PositionService.java

```diff
@@ -79,8 +79,10 @@
     public boolean isLocked();
 
     /**
-     * Returns the last sentence received from the gps.
+     * @deprecated
+     *             Returns the last sentence received from the gps.
```

Comment:
Do we have an API method to use instead? If so, maybe let's reference it here.

Reply:
Le's add also from when the API has been deprecated. It will help also with removals

Reply:
> Do we have an API method to use instead? If so, maybe let's reference it here.

No, the idea is to now allow to access directly to the underlying position provider.

Reply:
> Le's add also from when the API has been deprecated. It will help also with removals

I've added since 2.2.0

Reply:
deprecated annotation removed